modules/claims_api/app/models/claims_api/validated_token.rb

```diff
@@ -26,37 +23,6 @@ def initialize(token_validation_url, token_string, audience)
 
     attr_reader :validated_token_data
 
-    def validate_jwt_values
-      pubkey = public_key
-      JWT.decode(@token_string, pubkey, true, algorithm: 'RS256')[0]
-    rescue JWT::ExpiredSignature => e
-      Rails.logger.info(e.message, token: @token_string)
-      raise error_klass(e.message)
-    rescue JWT::DecodeError => e
-      raise error_klass(e.message)
-    end
-
-    def public_key
-      decoded_token = JWT.decode(@token_string, nil, false, algorithm: 'RS256')
-      iss = decoded_token[0]['iss']
-      kid = decoded_token[1]['kid']
-      key = OIDC::KeyService.get_key(kid, iss)
-      if key.blank?
-        StatsD.increment('okta_kid_lookup_failure', 1, tags: ["kid:#{kid}"])
-        Rails.logger.info('Public key not found', kid:, exp: decoded_token[0]['exp'])
-        raise error_klass("Public key not found for kid specified in token: '#{kid}'")
-      end
-
-      key
-    rescue JWT::DecodeError => e
-      raise error_klass("Unable to determine public key: #{e.message}")
-    end
-
-    def error_klass(error_detail_string)
-      # If there is an error validating the token, raise it here
-      ::Common::Exceptions::TokenValidationError.new(detail: error_detail_string)
-    end
-
```

Comment:
@crolarlibertyva - I believe this was added back as our consumers reported a change in the response for v1. @rockwellwindsor-va can probably provide more details but we were seeing a 401 unauthorized, but it was reported that v1 (before the changes we introduced a month or so ago) sent back a "Signature has expired." message. If we're looking at this as the new intent, it's fine but we would need to relay that to consumers before making this change.

Reply:
From our consumer:

```
Here is a sample error response from Aug 30:

{
"errors": [

{ "title": "Not authorized", "detail": "Signature has expired", "code": "401", "status": "401" }
]
}

Here is a sample error response from Aug 31:

{
"errors": [

{ "title": "Not authorized", "detail": "Not authorized", "code": "401", "status": "401" }
]
}
```

Reply:
Let me look into processing the error response from the validation call to pass along the error detail to indicate 'signature  expired', and other items like "invalid audience".

Reply:
@FonzMP : Updated and confirmed the response to be 
```
{
  "errors": [
    {
      "title": "Not authorized",
      "detail": "Signature has expired",
      "code": "401",
      "status": "401"
    }
  ]
}
```

Reply:
So the original response is now intact in the case of an expired token.